src/frontend/planner_test/tests/testdata/input/generate_series_with_now.yaml

```diff
@@ -0,0 +1,30 @@
+- sql: |
+    select * from generate_series(
+      '2024-06-21 17:36:00'::timestamptz,
+      now(),
+      interval '1 hour'
+    );
+  expected_outputs:
+    - logical_plan
+    - optimized_logical_plan_for_stream
+    - stream_plan
+- sql: |
+    select * from generate_series(
+      '2024-06-21 17:36:00'::timestamp, -- `timestamp` type is not supported
+      now(),
+      interval '1 hour'
+    );
+  expected_outputs:
+    - binder_error
+- sql: |
+    select * from generate_series(
+      now() - interval '1 hour',
```

Comment:
Interesting. Shouldn't this be able to fold into constant?

Reply:
Then the `now()` can be even more confusing🤣

Reply:
Maybe we need something like `session_now()`.